receiver/zipkinreceiver/trace_receiver.go

```diff
@@ -99,12 +100,14 @@ func (zr *ZipkinReceiver) Start(ctx context.Context, host component.Host) error
 		var listener net.Listener
 		listener, err = zr.config.HTTPServerSettings.ToListener()
 		if err != nil {
-			host.ReportFatalError(err)
 			return
 		}
+		zr.shutdownCh = make(chan struct{})
 		go func() {
+			defer close(zr.shutdownCh)
+
 			err = zr.server.Serve(listener)
-			if err != nil {
+			if err != nil && err != http.ErrServerClosed {
```

Comment:
Actually it is an "and" so just unnecessary extra protection.

Reply:
Yeah, will remove the check for error `nil` for HTTP server. For grpc the docs imply that it can actually return nil: "Serve will return a non-nil error unless Stop or GracefulStop is called." Looking at the sources it can definitely return nil.

Reply:
done